### DISCUSSION TITLE
Can't save any settings?!

### DISCUSSION THREAD

**Discussion by collin80** [WerWolv/ImHex#1776] [Get Help] [answered]
I'm on Ubuntu 24.04 and running ImHex 1.35.1. I used the .deb to install it directly into Ubuntu rather than using an appimage (though this behavior is the same both ways).

I can't save any settings at all. If I change them then they do not persist upon restarting the application. It also always tells me it's the first time I'm running it. So, something isn't saving configuration properly but I'm not sure why. Any tips?

**Comment by WerWolv** [comment, marked as answer]
Please update to the latest release of ImHex. v1.35.3 has this issue fixed.
